<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="boardGraphicsView">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>20</y>
      <width>661</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="handGraphicsView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>661</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="playerGraphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>410</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="endGameButton">
       <property name="text">
        <string>End Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playGameButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="advanceTurnButton">
       <property name="text">
        <string>Advance Turn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startOverButton">
       <property name="text">
        <string>Start Over</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>223</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="numberOfPlayersLabel">
       <property name="text">
        <string>Number of Players</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="numberOfPlayersComboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
